app/controllers/course/admin/admin_controller.rb

```diff
@@ -11,8 +11,28 @@ def update #:nodoc:
     end
   end
 
+  def destroy #:nodoc:
+    if current_course.destroy
+      delete_success
+    else
+      delete_failure
+    end
+  end
+
   private
 
+  def delete_success #:nodoc:
```

Comment:
and these are after the params -- params are the first private methods declared.